src/lwc/pubsubNoPageRef/pubsubNoPageRef.html

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
Do we know if we need the template to be included even when its blank?

Reply:
Unfortunately I believe the HTML file is required